sql/core/src/main/scala/org/apache/spark/sql/streaming/StreamingQueryManager.scala

```diff
@@ -219,10 +219,11 @@ class StreamingQueryManager private[sql] (sparkSession: SparkSession) {
       }
 
       // If offsets have already been created, we trying to resume a query.
-      if (!recoverFromCheckpointLocation) {
+      if (recoverFromCheckpointLocation) {
         val checkpointPath = new Path(checkpointLocation, "offsets")
         val fs = checkpointPath.getFileSystem(df.sparkSession.sessionState.newHadoopConf())
         if (fs.exists(checkpointPath)) {
+          // Currently, offset recovery from checkpoint is not supported.
```

Comment:
Maybe I understood the code path wrongly. I thought that the recovery will be done if fs.exists(checkpointPath) is true. However, it will only throw the exception, so I thought that offset recovery from checkpoint is not supported. Am I missing something?